examples/step-55/step-55.cc

```diff
@@ -456,7 +456,8 @@ namespace Step55
         dof_handler, coupling, dsp, constraints, false);
       SparsityTools::distribute_sparsity_pattern(
         dsp,
-        dof_handler.compute_locally_owned_dofs_per_processor(),
+        Utilities::MPI::all_gather(mpi_communicator,
+                                   dof_handler.locally_owned_dofs()),
```

Comment:
This is unrelated to this patch, but I believe we should replace this argument by simply `dof_handler.locally_owned_dofs()` - I must have forgotten those in #9710.